### ISSUE TITLE
Provide handling when user does not have writing permissions to any of the entity collections

### ISSUE THREAD

**Issue by ilovan** [cwrc/cwrc-entity-management-forms-static#59]
## Expected Behaviour
"Create New" should be grayed out and inactive on th entity lookup for users who do not have writing permissions for any of the entity collections.

**Comment by ajmacdonald** [comment]
https://github.com/cwrc/cwrc-entity-management-forms-static/commit/75f3e4989c83784dfd46a33272d9df98b17d0f48

**Comment by ajmacdonald** [comment]
It looks like it's possible to edit an existing entity even if a user does not have permission to create new entities. Is this intended?

**Comment by ajmacdonald** [comment]
https://github.com/cwrc/CWRC-PublicEntityDialogs/commit/778cd5fc3da4599a9afc14a2701de79a2646897d

**Comment by ilovan** [comment]
you are right! this is a major use. can you fix it according to this comment from Brad, or do we need to ask him? https://github.com/cwrc/cwrc-entity-management-forms-static/issues/52#issuecomment-433433846

**Comment by ajmacdonald** [comment]
I'll need his help with this one.

**Comment by ilovan** [comment]
current issue confirmed fixed on dev-02. 
separate issue created for entity editing behaviour at https://github.com/cwrc/cwrc-entity-management-forms-static/issues/61
